metanode/manager_op.go

```diff
@@ -212,7 +212,7 @@ func (m *metadataManager) opCreateInode(conn net.Conn, p *Packet,
 		return
 	}
 
-	err = mp.CreateInode(req, p)
+	err = mp.CreateInode(req, p, remoteAddr)
```

Comment:
why not record audit log in opCreateInode directly?

Reply:
We need to know the result of operation(new inode number), so it need to record in `CreateInode`.

Reply:
other func, eg createDentry, unlinkInode and so on, are they necessary to record inner?

Reply:
and for some operations, eg appendExtent, trunc, it may be better to print total req param json msg in audit log direct, which is helpful to locate problem.

Reply:
Of course, you have done a great job, and this version is sufficient for use.

Reply:
> other func, eg createDentry, unlinkInode and so on, are they necessary to record inner?

Yes, they is unnecessary.